lib/mix/lib/mix/tasks/profile.eprof.ex

```diff
@@ -229,7 +255,13 @@ defmodule Mix.Tasks.Profile.Eprof do
   defp format_total(total_time, total_count) do
     time_per_call = :erlang.float_to_binary(divide(total_time, total_count), [{:decimals, 2}])
 
-    ["Total", Integer.to_string(total_count), "100.00", Integer.to_string(total_time), time_per_call]
+    [
+      "Total",
+      Integer.to_string(total_count),
```

Comment:
Let's move those string conversions to the previous line so everything fits a single line here too.